default.json

```diff
@@ -56,8 +72,12 @@
         "mcr.microsoft.com/dotnet/aspnet",
         "mcr.microsoft.com/dotnet/runtime",
         "mcr.microsoft.com/dotnet/runtime-deps",
+        "Azure.Identity",
         "/^[mM]icrosoft\\./",
-        "/^[sS]ystem\\./"
+        "/^[sS]ystem\\./",
+        "coverlet.collector",

```

Comment:
(nothing to change in this PR, just a question)
I'm curious about our usage of coverlet. Do we actually use it to compute code coverage, or it's in our projects because it's part of the template? I see many examples of the latter.

Reply:
I created an improvement story for that: IDP-2283

Reply:
BTW, if we want to add code coverage to our projects, this library is not needed anymore. Since .NET SDK 8, code coverage is builtin: https://www.meziantou.net/computing-code-coverage-for-a-dotnet-project.htm (`dotnet test --collect:"Code Coverage"`)